biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -291,7 +291,7 @@ subroutine LoggingMortality_frac( pft_i, dbh, canopy_layer, lmort_direct, &
          ! transfer of area to secondary land is based on overall area affected, not just logged crown area
          ! l_degrad accounts for the affected area between logged crowns
          if(prt_params%woody(pft_i) == itrue)then ! only set logging rates for trees
-            if (cur_harvest_tag == 0) then
+            if (cur_harvest_tag == 0 .or. cur_harvest_tag == 2) then
```

Comment:
HI @sshu88  -- would it be possible to replace these integers that you are comparing `cur_harvest_tag ` to, here and elsewhere, with named integer constants that describe more clearly what these conditions refer to?

Reply:
Nice suggestion! I updated the code.